recipes/folly/all/patches/0016-find-packages.patch

```diff
@@ -12,12 +12,19 @@ index e0e02c0..2bd4940 100644
  list(APPEND FOLLY_INCLUDE_DIRECTORIES ${DOUBLE_CONVERSION_INCLUDE_DIR})
  
 -find_package(Gflags MODULE)
+-set(FOLLY_HAVE_LIBGFLAGS ${LIBGFLAGS_FOUND})
+-if(LIBGFLAGS_FOUND)
+-  list(APPEND FOLLY_LINK_LIBRARIES ${LIBGFLAGS_LIBRARY})
+-  list(APPEND FOLLY_INCLUDE_DIRECTORIES ${LIBGFLAGS_INCLUDE_DIR})
+-  set(FOLLY_LIBGFLAGS_LIBRARY ${LIBGFLAGS_LIBRARY})
+-  set(FOLLY_LIBGFLAGS_INCLUDE ${LIBGFLAGS_INCLUDE_DIR})
 +find_package(gflags MODULE)
- set(FOLLY_HAVE_LIBGFLAGS ${LIBGFLAGS_FOUND})
- if(LIBGFLAGS_FOUND)
-   list(APPEND FOLLY_LINK_LIBRARIES ${LIBGFLAGS_LIBRARY})
-@@ -61,12 +61,12 @@ if(LIBGFLAGS_FOUND)
-   set(FOLLY_LIBGFLAGS_INCLUDE ${LIBGFLAGS_INCLUDE_DIR})
++set(FOLLY_HAVE_LIBGFLAGS ${GFLAGS_FOUND})
++if(GFLAGS_FOUND)
```

Comment:
why `GFLAGS_FOUND` and not `gflags_FOUND`?

Reply:
Yeah, it should be gflags_FOUND, but GFLAGS_FOUND also work when I tested it yesterday. No matter why, I have changed it to gflags_FOUND.

Reply:
> GFLAGS_FOUND also work when I tested it yesterday

It may work depending on case sensitivity of the system on which you have tested the build, but it's not robust.